#This message allows the robot to send it's core information
pose robot_pose
position ball_position
int16 imu_value
uint8 has_ball
bool sees_ball
position[] obstacles
